docs/contributing/README.md

```diff
@@ -0,0 +1,9 @@
+# How to contribute?
+
+We have an [extensive contribution guide](docs/contributing/CONTRIBUTING.md) outlining all the steps to contribute, but if you just want to start getting involved, follow the next two steps:
+
+1. Look at available work on [app.dework.xyz/pokt-network](https://app.dework.xyz/pokt-network) or look at our [starter tasks](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22starter+task%22) / [community](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22community%22+) tasks on Github.
+
+2. Reach out to the team on Discord in the [#v1-dev channel](https://discord.com/channels/553741558869131266/986789914379186226) or [#v1-research channel](https://discord.com/channels/553741558869131266/936666517498650644).
```

Comment:
Not sure if by design but #v1-dev on discord is read-only, at least for my user 🙂

Reply:
Lol, it is.
<img width="148" alt="Screen Shot 2022-08-05 at 10 17 56 PM" src="https://user-images.githubusercontent.com/1892194/183229829-773f670d-398b-4d74-bb55-3a6ef630717c.png">

Try again :)